queries/dart.scm

```diff
@@ -16,3 +16,14 @@
   (set_or_map_literal)
   (map_pattern)
 ] @map
+
+;;!! class Foo {}
+;;!  ^^^^^^^^^^^^
+(
+  [
+    (class_definition
+
```

Comment:
this empty line shouldn't appear. Have you installed all the recommended extensions? In particular, Andreas's vscode extension should autoformat `.scm` files. Though maybe you need to configure it to be the formatter for `.scm` files?